__tests__/comment.spec.ts

```diff
@@ -211,17 +209,63 @@ describe('handleComment', () => {
       .get(`/repos/${owner}/${repo}/issues/${prIssue}/comments?page=1&per_page=2`)
       .reply(200, [getCommentsResponse[0]]);
 
+    nock('https://api.github.com')
+      .patch(`/repos/${owner}/${repo}/issues/comments/1`)
+      .reply(200, [getCommentsResponse[0]]);
+
     const response = await handleComment(client, {
       owner,
       repo,
       prNumber: 1,
-      matchingRules: [{ ...rule, customMessage: 'first comment' }],
+      matchingRules: [{ ...rule, customMessage: 'new comment that should be updated' }],
       base: '',
       sha: '',
       files: [],
       rules: [],
     });
 
-    expect(response).toMatchInlineSnapshot('Array []');
+    expect(response).toMatchInlineSnapshot(`
+      Array [
+        Object {
+          "data": Array [
+            Object {
+              "body": "<!-- USE_HERALD_ACTION rules/rule.json -->
```

Comment:
what would happen when we get 2 rules in there? can you have this test have 2 rules instead so we can see that?